doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,245 @@
+| ceph_os_user | String | Yes | the user of ceph OS. |
+| ceph_os_password | String | Yes | the user's password of ceph OS. |
+| ceph_manage_plane_float_ip | String | Yes |  |
+| ceph_network_manage_plane_float_ip | String | Yes |  |
+| zms_network_manage_admin_password | String | Yes |  |
+| ceph_device_name | String | Yes |  |
+| ceph_iscsi_device_name | String | Yes |  |
+| glance_pool_name | String | Yes |  |
+| nova_pool_name | String | Yes |  |
+| belong_to_az_one | String | Yes |  |
+| cinder_infos | List | Yes |a list of cinder infos which are cinder pool name and backend info |
+
+<p align="center"><b>Table 5-3-8-2:</b> Cluster information.</p>
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cinder_pool_name | String | Yes | cinder pool name|
+| cinder_backend_name | String | Yes | backend name cinder connected|
+<p align="center"><b>Table 5-3-8-3:</b> Cinder information.</p>
+
+### 5.3.9 NTP server informations
+primary and backup NTP server information.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| master_server_ip | String | Yes | |
+| master_server_timezone | String | Yes | |
+| backup_server_ip | String | No | |
+| backup_server_timezone | String | No | |
+
+<p align="center"><b>Table 5-3-9:</b> NTP server information.</p>
+
+### 5.3.10 DNS server informations
+DNS server informmation if VIM deployment requires.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| external_dns_network | String | No | DNS information if VIM deployment needed. |
+| external_dns_ip | String | No | |
+
+<p align="center"><b>Table 5-3-10:</b> DNS server information.</p>
+
+### 5.3.10 Deployment host informations
+Deployment host setting, which should have the access for the openstack nodes network.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ip_address | String | No | Ip address of debug PC |
+| mask | String | No | mask of debug PC|
+| gateway | String | No |gateway of debug PC |
+
+<p align="center"><b>Table 5-3-10:</b> Deployment host information.</p>
+
+### 5.3.11 Deployment control informations
```

Comment:
This should be 5.3.12. Same for the table in this section.

Reply:
yes, thanks, check the latest please.